docs/development.rst

```diff
@@ -94,32 +94,46 @@ Usage::
 Creating a new release
 ----------------------
 
-Checklist::
-
-- all issues for this milestone closed?
-- any low hanging fruit left on the issue tracker?
-- run tox on all supported platforms via vagrant, check for test fails.
-- is Travis CI happy also?
-- update CHANGES.rst (compare to git log). check version number of upcoming release.
-- check MANIFEST.in and setup.py - are they complete?
+Checklist:
+
+- make sure all issues for this milestone are closed or move to the
+  next milestone
+- look and fix any low hanging fruit left on the issue tracker
+- run tox on all supported platforms via vagrant, check for test failures
+- check that Travis CI is also happy
+- update ``CHANGES.rst``, based on ``git log $PREVIOUS_RELEASE..``
+- check version number of upcoming release in ``CHANGES.rst``
+- verify that ``MANIFEST.in`` and ``setup.py`` are complete
 - tag the release::
 
-  git tag -s -m "tagged release" 0.26.0
+    git tag -s -m "tagged release" 0.26.0
 
-- cd docs ; make html  # to update the usage include files
-- update website with the html
-- create a release on PyPi::
+- update usage include files::
```

Comment:
i'll let you complete that one as I don't quite understand how to update the website... it would be nice to avoid duplication with the above docs instructions, true...